src/sfctl/custom_cluster.py

```diff
@@ -39,6 +40,15 @@ def select_arg_verify(endpoint, cert, key, pem, ca, aad, no_verify): #pylint: di
     if pem and any([cert, key]):
         raise CLIError(usage)
 
+def show_connection():
```

Comment:
I don't like the idea of printing an en-us string for this. Can we just make it return None or empty string?